### ISSUE TITLE
Error when Convert 3DGS to Mesh with NeRF and Marching Cubes

### ISSUE THREAD

**Issue by Nziner** [MrForExample/ComfyUI-3D-Pack#35]
Hi! Finally able to install (not from the first time).
Workflow "Using_Stack_Orbit_Camera_Poses_to_conditioning_StableZero123.json" works. But the "Triplane_Gaussian_Transformers_to_3DGS_and_3DMesh.json" shows an error at the final stage of converting gaussian points to mesh.

```
Error occurred when executing [Comfy3D] Convert 3DGS to Mesh with NeRF and Marching Cubes:

Ninja is required to load C++ extensions

File "D:\Work\AI\StableDiffusion\ComfyUI_portable\ComfyUI\execution.py", line 152, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\ComfyUI\execution.py", line 82, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\ComfyUI\execution.py", line 75, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\ComfyUI\custom_nodes\ComfyUI-3D-Pack\nodes.py", line 1336, in convert_gs_ply
converter = GSConverterNeRFMarchingCubes(config_defaults[gs_config], gs_ply).cuda()
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\ComfyUI\custom_nodes\ComfyUI-3D-Pack\algorithms\nerf_marching_cubes_converter.py", line 27, in init
from kiui.gridencoder import GridEncoder
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\python_miniconda_env\ComfyUI\Lib\site-packages\kiui\gridencoder\__init__.py", line 1, in
from .grid import GridEncoder
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\python_miniconda_env\ComfyUI\Lib\site-packages\kiui\gridencoder\grid.py", line 9, in
from .backend import _backend
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\python_miniconda_env\ComfyUI\Lib\site-packages\kiui\gridencoder\backend.py", line 63, in
_backend = load(
^^^^^
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\python_miniconda_env\ComfyUI\Lib\site-packages\torch\utils\cpp_extension.py", line 1306, in load
return _jit_compile(
^^^^^^^^^^^^^
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\python_miniconda_env\ComfyUI\Lib\site-packages\torch\utils\cpp_extension.py", line 1710, in _jit_compile
_write_ninja_file_and_build_library(
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\python_miniconda_env\ComfyUI\Lib\site-packages\torch\utils\cpp_extension.py", line 1793, in _write_ninja_file_and_build_library
verify_ninja_availability()
File "D:\Work\AI\StableDiffusion\ComfyUI_portable\python_miniconda_env\ComfyUI\Lib\site-packages\torch\utils\cpp_extension.py", line 1842, in verify_ninja_availability
raise RuntimeError("Ninja is required to load C++ extensions")
```

**Comment by MrForExample** [comment]
From the error, it seems you are missing Ninja package, try following command in your conda environment:
`pip install --upgrade ninja`

**Comment by Nziner** [comment]
If only it were that simple :)
This is what I tried to do first, the thing is that the folder exists, but for some reason is not used. 

I tried:
1. A complete reinstallation of "python_miniconda_env"
2. conda install ninja
3. pip uninstall ninja
pip install ninja==1.10.2

I think the problem is in path and windows can't find the correct path.

**Comment by Nziner** [comment]
Well, I've solved the problem:

1. First I downloaded and replaced ninja.exe from https://ninja-build.org. The process went on, but other errors appeared.
2. I realized that I was using a different version of Cuda.
"conda install -c "nvidia/label/cuda-12.1.0" cuda-toolkit" did not solve the problem, nor did setting PATH in Windows variables. 
3. Installed globally https://developer.nvidia.com/cuda-12-1-0-download-archive and the conversion process started, yay! 
4. But at the very end another error "No module named 'sklearn'" appeared. I rechecked the installation of "requirements.txt" and "requirements_post.txt". The error did not go away. I solved it with the command "pip install scikit-learn". 

After step 4 everything successfully started and worked.
